awx/ui_next/src/screens/User/UserOrganizations/UserOrganizationListItem.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { withI18n } from '@lingui/react';
+import { t } from '@lingui/macro';
+import {
+  DataList,
+  DataListItemCells,
+  DataListItemRow,
+  DataListItem,
+  DataListCell,
+} from '@patternfly/react-core';
+
+function UserOrganizationListItem({ organization, i18n }) {
+  return (
+    <DataList aria-label={i18n._(t`User Organization List Item`)}>
```

Comment:
This `DataList` needs to live in the parent component (PaginatedDataList should do this already)... structured the way it is produces multiple single-item lists (`<ul>`s) instead of one list with multiple items.